generators/delimited/testing/start_test

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# NOTE: this file (and 'delimited') use relative paths. Make sure to 
+# run this script from origins_generators/generators/delimited/.
+
+# Generate new output and compare it with the expected output file. 
+# A diff will not work because the facts may not be generated in 
+# the same order each time. (generator uses dictionaries)
+rm testing/test_output.csv
+./delimited testing/test_input.csv >> testing/test_output.csv
```

Comment:
The fix is to sort the output file ahead of time. So you only need to sort the generated facts. Note that the fact header will be sorted as well.